### DISCUSSION TITLE
What actually is the special sauce with Valve's kernel?

### DISCUSSION THREAD

**Discussion by Krutonium** [Jovian-Experiments/Jovian-NixOS#306] [Q&A] [answered]
Basically what I'm asking is why do we use Valve's kernel; what's stopping us from using the latest official kernel build from NixOS?

**Comment by K900** [comment, marked as answer]
There are critical drivers for the OLED models' display hardware, and a lot of AMDGPU backports, and some power usage tuning, and, and, and. Mainline will be usable on LCD models.

**Reply by Krutonium** [reply]
Excellent to know, thank you!
